resources/helm-broker/values.yaml

```diff
@@ -22,15 +22,21 @@ config:
 
       etcd:
         endpoints:
-          - http://core-helm-broker-etcd-stateful-client.kyma-system.svc.cluster.local:2379
+          - http://helm-broker-etcd-stateful-client.kyma-system.svc.cluster.local:2379
     - driver: memory
       provide:
         chart: ~
         bundle: ~
 
-
-etcd:
-  # By default etcd subchart will be release with name "core-etcd", and it's to much generic
-  # but currently there is no way to pass to the etcd subchart a parent Release.Name, see https://github.com/kubernetes/helm/issues/2506
-  # so to make it more verbose I've added nameOverride and now the release name will be "core-helm-broker-etcd"
-  nameOverride: "helm-broker-etcd"
\ No newline at end of file
+global:
+  containerRegistry:
+    path: eu.gcr.io/kyma-project
+  istio:
```

Comment:
we do not use istio right now, please remove

Reply:
Fixed